File: src/main/java/com/p2p/lending/dao/DeptDao.java
package com.p2p.lending.dao;

import java.util.List;

import com.p2p.lending.entity.Dept;

public interface DeptDao {

	List<Dept> findall();

	Dept findbyid(Integer did);

	int insert(Dept dept);

	int del(Integer did);

	int upd(Dept dept);
}

File: src/main/java/com/p2p/lending/dao/DeptDao.kt
package com.p2p.lending.dao

import com.p2p.lending.entity.Dept

interface DeptDao {

    fun findall(): List<Dept>

    fun findbyid(did: Int?): Dept

    fun insert(dept: Dept): Int

    fun del(did: Int?): Int

    fun upd(dept: Dept): Int
}